tests/integration/navigation/001-site-nav.spec.js

```diff
@@ -0,0 +1,37 @@
+const { test, expect } = require("@playwright/test");
+const waitForImagesToLoad = require("../../wait-for-images.js");
+
+const MOBILE_VIEWPORT = { width: 360, height: 800 };
+
+test.describe("Main site primary nav", () => {
+  test(`Primary nav (mobile)`, async ({ page }) => {
+    const PAGE_URL = `http://localhost:8000/en/`;
```

Comment:
This `localhost:8000/en` value seems to be repeated throughout our integration tests. It would be nice if it could be defined in a single place, like a `constants` file or something. Same thing for `mozfest.localhost:8000`... Not a block though, just an observation

Reply:
I like this idea! I'll create a new ticket for this as there are multiple places in the existing code that can benefit from this.

Reply:
Awesome, thanks!